Scripts/Projectile.cs

```diff
@@ -31,39 +33,61 @@ public Projectile(float range, float speed, Vector3 direction, PackedScene proje
 
     public override void _Ready()
     {
-        Area projectileInstance = projectileScene.Instance() as Area;
-        projectileLength = projectileInstance.Scale.z;
-
+        RigidBody projectileInstance = projectileScene.Instance() as RigidBody;
+        // Area projectileInstance = projectileScene.Instance() as Area;
+        // projectileLength = projectileInstance.Scale.z;
+        projectileLength = 3;
         projectileInstance.Connect("body_entered", this, "OnBodyEnteredProjectile");
-
+        projectileInstance.GravityScale = 2;
```

Comment:
Why?

Reply:
Don't want the projectile to fly too far. When projectile reaches the ground it will destroy itself. Can change it to a lifetime later.